tests/tests-hz38/src/main/java/com/hazelcast/simulator/tests/quorum/QuorumMapTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.simulator.tests.quorum;
+
+import org.junit.Assert;
+
+import com.hazelcast.core.IMap;
+import com.hazelcast.quorum.QuorumException;
+import com.hazelcast.simulator.test.AbstractTest;
+import com.hazelcast.simulator.test.BaseThreadState;
+import com.hazelcast.simulator.test.annotations.Setup;
+import com.hazelcast.simulator.test.annotations.TimeStep;
+
+public class QuorumMapTest extends AbstractTest {
+
+    // properties
+    public int keyCount = 100;
+    public int gracePeriodInMilliSec = 2000;
+    private volatile long lastInconsistencyTimestamp = 0L;
```

Comment:
Add one enter to separate the public properties from the non public regular fields.